src/main/java/com/stripe/model/Account.java

```diff
@@ -810,6 +810,15 @@ public CapabilityCollection capabilities(AccountCapabilitiesParams params, Reque
     @SerializedName("p24_payments")
     String p24Payments;
 
+    /**
+     * The status of the paynow payments capability of the account, or whether the account can
+     * directly process paynow charges.
+     *
+     * <p>One of {@code active}, {@code inactive}, or {@code pending}.
+     */
+    @SerializedName("paynow_payments")
+    String paynowPayments;
+
```

Comment:
you're missing `us_bank_account_ach_payments` which is part of the US Bank Account PM launch